x-pack/solutions/search/plugins/search_homepage/public/components/search_homepage.tsx

```diff
@@ -24,7 +24,7 @@ export const SearchHomepagePage = () => {
 
   return (
```

Comment:
Am I missing the redirect hook for sending the user to the start page when they have no indices? I think it should be added to this file. but if it's somewhere else and I missed it let me know.